### PR TITLE
Adds a lot of stuff for the technomancer, burned out cell and energy shotgun

### PR DESCRIPTION
**Pull request by Smokingsquirrel**
## About The Pull Request
OK, so where do i start, the fuck. 
Easy things first, all sprites have been done by Armolitskiy on Discord. Code is done by me.

Adds yellow goggles and a technomancer jumpsuit, long overdue.
Technomancers also got a branded weapon, a railgun and a pistol. The pistol is coded but currently not obtainable aside from adminspawn. The gun should be printable in an autolathe with the correct disk. I think. If not, well, then the gun is just not available as of yet and will be implemented by code on a later date. It was enough work getting all the sprites and codes done.

This PR also adds in the Sprites requested by R0B0, namely a burnt out cell and an energy shotgun. Both are just sprites as of now and uncoded.

Untitled

## Why It's Good For The Game
New guns and long overdue techno content.

## Changelog
:cl:
add: Added new technomnacer suit and railguns
/:cl:

### CODE REVIEW

**Comment by SpadesSlick** [comment]
Separate changelog into different parts, for different things included to a game.

Guns must be obtainable. Include them on disc and provide one of them to TE.

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -19,6 +19,55 @@
...
+	slot_flags = SLOT_BELT|SLOT_BACK
+	w_class = ITEM_SIZE_NORMAL
+	force = WEAPON_FORCE_PAINFUL
+	origin_tech = list(TECH_COMBAT = 4, TECH_MAGNET = 4)
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_STEEL = 8, MATERIAL_SILVER = 10)
+	fire_delay = 14
+	charge_cost = 100
+	recoil_buildup = 1 //pulse weapons have a bit more recoil
+	price_tag = 3500
+	projectile_type = /obj/item/projectile/beam/midlaser
```
Comment (SpadesSlick) [inline code/modules/projectiles/guns/energy/laser.dm:39]: How come that railgun shoots lasers? This must be chaged.

File: code/datums/autolathe/guns.dm [hunk 2]
```diff
@@ -204,6 +204,10 @@
 	name = "NT LG \"Lightfall\""
 	build_path = /obj/item/weapon/gun/energy/laser
 
+/datum/design/autolathe/gun/laser/railgun
```
Comment (SpadesSlick) [inline code/datums/autolathe/guns.dm:207]: You need to create a disc with appropriate design for it to work.

**Comment by MadmanMartian** [comment]
>This PR also adds in the Sprites requested by R0B0, namely a burnt out cell and an energy shotgun. Both are just sprites as of now and uncoded.

My dude, I'm already using them in #4969

Also stop closing PRs the moment you get review comments, you can just push to the same branch and it'll update the PR!

**Pull request closed without merging**